text/0097-log-data-model.md

```diff
@@ -0,0 +1,1340 @@
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>%t</td>
+    <td>Timestamp</td>
+    <td>Time when an event occurred measured by the origin clock.</td>
+    <td>Timestamp</td>
+  </tr>
+  <tr>
+    <td>%a</td>
+    <td>string</td>
+    <td>Client IP</td>
+    <td>Attributes["net.peer.ip"]</td>
+  </tr>
+  <tr>
+    <td>%A</td>
+    <td>string</td>
+    <td>Server IP</td>
+    <td>Attributes["net.host.ip"]</td>
+  </tr>
+  <tr>
+    <td>%h</td>
+    <td>string</td>
+    <td>Remote hostname. </td>
+    <td>Attributes["net.peer.name"]</td>
+  </tr>
+  <tr>
+    <td>%m</td>
+    <td>string</td>
+    <td>The request method.</td>
+    <td>Attributes["http.method"]</td>
+  </tr>
+  <tr>
+    <td>%v,%p,%U,%q</td>
+    <td>string</td>
+    <td>Multiple fields that can be composed into URL.</td>
+    <td>Attributes["http.url"]</td>
+  </tr>
+  <tr>
+    <td>%>s</td>
+    <td>string</td>
+    <td>Response status.</td>
+    <td>Attributes["http.status_code"]</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>any</td>
+    <td>Structured data.</td>
+    <td>Attributes, key=TBD</td>
+  </tr>
+</table>
+
+### CloudTrail Log Event
+
+<table>
+  <tr>
+    <td>Field</td>
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>eventTime</td>
+    <td>string</td>
+    <td>The date and time the request was made, in coordinated universal time (UTC).</td>
+    <td>Timestamp</td>
+  </tr>
+  <tr>
+    <td>eventSource</td>
+    <td>string</td>
+    <td>The service that the request was made to. This name is typically a short form of the service name without spaces plus .amazonaws.com.</td>
+    <td>Resource["service.name"]?</td>
+  </tr>
+  <tr>
+    <td>awsRegion</td>
+    <td>string</td>
+    <td>The AWS region that the request was made to, such as us-east-2.</td>
+    <td>Resource["cloud.region"]</td>
+  </tr>
+  <tr>
+    <td>sourceIPAddress</td>
+    <td>string</td>
+    <td>The IP address that the request was made from.</td>
+    <td>Resource["net.peer.ip"] or Resource["net.host.ip"]? TBD</td>
+  </tr>
+  <tr>
+    <td>errorCode</td>
+    <td>string</td>
+    <td>The AWS service error if the request returns an error.</td>
+    <td>ShortName</td>
+  </tr>
+  <tr>
+    <td>errorMessage</td>
+    <td>string</td>
+    <td>If the request returns an error, the description of the error.</td>
+    <td>Body</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>*</td>
+    <td></td>
+    <td>Attributes["cloudtrail.*"]</td>
+  </tr>
+</table>
+
+### Google Cloud Logging
+
+Field            | Type               | Description                                             | Maps to Unified Model Field
+-----------------|--------------------| ------------------------------------------------------- | ---------------------------
+timestamp        | string             | The time the event described by the log entry occurred. | Timestamp
```

Comment:
(Duping the comment I posted on the timestamp field) - Google Cloud Logging also has an optional insert id field complementing the timestamp that is used for de-duping and sometimes implicitly for tie-breaking in ordering.
See: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry

Reply:
This is captured as a separate issue: https://github.com/open-telemetry/opentelemetry-specification/issues/597